.ci/vs2022.yml

```diff
@@ -283,6 +283,11 @@ stages:
               filename: $(VsDevCmd)
               arguments: -no_logo -arch=$(arch) -host_arch=amd64 -vcvars_ver=14.31
               modifyEnvironment: true
+          - task: UsePythonVersion@0
+            inputs:
+              versionSpec: '3.10' 
+              addToPath: true 
+              architecture: 'x64'
```

Comment:
Actually, I think that this needs to be `$(arch)`.  We also have the ARM64 toolchain.

Reply:
Oh, ARM64 would do fine. It is not even broken in main branch. We are using pythonarm64 package (with `EXTRA_CMAKE_ARGS` wiring it to the config), so ARM64 build is immune to any other Python installed on the build machine. It would be used as interpreter for codegen tasks, but lldb integration is using the correct arch.

I think I could make this task run only for x64 builds, because ARM64 doesn't need any python adjustment at the moment.

Unfortunately, `UsePythonVersion` task only supports x86 and x64. We could've implement a simple and common way to set up Python for all archs in such case.